compiler/onecc-docker/debian/onecc-docker.install

```diff
@@ -0,0 +1,2 @@
+/onecc-docker /usr/share/one/bin/
+/docker/Dockerfile /usr/share/one/docker/
```

Comment:
```suggestion
# TODO Change the file location
/onecc-docker /usr/share/one/bin/
/docker/Dockerfile /usr/share/one/docker/
```

In the future, in the process of building one compiler, this tools will move to the cmake output directory and then a debian package will be created. So I added `TODO`.